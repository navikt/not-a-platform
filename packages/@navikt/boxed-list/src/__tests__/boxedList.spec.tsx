import * as React from 'react';
import renderer from 'react-test-renderer';
import BoxedList from '../index';

test('Should render', () => {
    const component = renderer.create(
        <BoxedList>
            <li>Test</li>
        </BoxedList>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
