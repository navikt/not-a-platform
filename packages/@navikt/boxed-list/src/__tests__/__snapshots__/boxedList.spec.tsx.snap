// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 1`] = `
<ul
  className="boxedList"
>
  <li>
    Test
  </li>
</ul>
`;
